### PR TITLE
feat(changelog): Added Changelog page

### LINKED ISSUE #68: CHANGELOG page
## Problem

The user can't know the type of changes that happened in the project without digging through PRs and commits. It would be useful to have a CHANGELOG page where we list changes on the website.

## Solution

To log changes, include a `CHANGELOG.txt` file where it contains the following information:
```
AUTHOR,YYYY-MM-DD
CHANGE_1
---
AUTHOR,YYYY-MM-DD
CHANGE_2
---
...
```
Here are some example values:
- `YYYY-MM-DD`: 2022-07-09
- `CHANGE_1`: Included changelog page `https://arbml.github.io/masader/changelog`
- `AUTHOR`: GITHUB_USERNAME

Update web service to read the `changelog.txt` file and do pre-processing to turn it to a json format for ease of reading and processing from the front-end side.

Update front-end codebase to fetch from the web service and display the information correctly.

### PR DESCRIPTION
**Pull request by sudomaze**
Resolves #68 #66

### CODE REVIEW

**Comment by sudomaze** [comment]
@AliOsm please feel free to add your contribution and others on `masader-service` repo to the `CHANGELOG.md`.

`0.5.0` = Re-designing the website
`0.7.0` = Moving to Jekyll
`0.8.0` = Improvements

Soon, we will have
`0.9.0` = New UI
`1.0.0` = Demo release

The version is somewhat isn't meaningful as we should have had patches and stuff, but we could start to have this enforced after `1.0.0` where there won't be any push to `main` and every PR will need to update the changelog accordingly.

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+### Added
+
+- Added icon(Favicons) [#6](https://github.com/ARBML/masader/issues/6) by [@MutlaqAldhbuiub](https://github.com/MutlaqAldhbuiub).
+- Added a new styles [#10](https://github.com/ARBML/masader/pull/10) by [@alonemazin](https://github.com/alonemazin).
+
+### Changed
+
+- Refactored the project structure [#7](https://github.com/ARBML/masader/pull/7) by [@MutlaqAldhbuiub](https://github.com/MutlaqAldhbuiub).
+- Redesigned the pages according to the org. brand [#7](https://github.com/ARBML/masader/pull/7 by [@MutlaqAldhbuiub](https://github.com/MutlaqAldhbuiub).
```
Comment (zaidalyafeai) [inline CHANGELOG.md:64]: missing closing parenthesis [#7](https://github.com/ARBML/masader/pull/7)

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+### Added
+
+- Included instructions in `README.md` and `CONTRIBUTING.md` [#48](https://github.com/ARBML/masader/pull/48) by [sudomaze](https://github.com/sudomaze).
+- Added a default value for `Graphs` page [#48](https://github.com/ARBML/masader/pull/48) by [sudomaze](https://github.com/sudomaze).
+
+### Changed
+
+- Moved to `jekyll` [#48](https://github.com/ARBML/masader/pull/48) by [sudomaze](https://github.com/sudomaze).
+- Updated `CONTRIBUTING.md` [#21](https://github.com/ARBML/masader/pull/21) by [unifoliatus](https://github.com/unifoliatus).
+- Refactored JS files for `jekyll` release [#48](https://github.com/ARBML/masader/pull/48) by [sudomaze](https://github.com/sudomaze) and [#33](https://github.com/ARBML/masader/pull/33 by [yousef337](https://github.com/yousef337).
```
Comment (zaidalyafeai) [inline CHANGELOG.md:43]: missing closing parenthesis [#33](https://github.com/ARBML/masader/pull/33)

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
+---
+permalink: /changelog
+layout: changelog
+---
+
+# Changelog
```
Comment (zaidalyafeai) [inline CHANGELOG.md:6]: I would say reduce the font size for headers.

File: CHANGELOG.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [0.8.0] - 2022-07-11
```
Comment (zaidalyafeai) [inline CHANGELOG.md:13]: On what grounds were the version numbers chosen? Why there is no 0.6.0?

Comment (sudomaze) [reply CHANGELOG.md:13]: I will have 0.5.0 to be 0.6.0

**Comment by zaidalyafeai** [comment]
Looks good to me. Leaving to @AliOsm if there are any comments.

**Review by zaidalyafeai — APPROVED** [review]
Looks, good!

**Pull request merged**